v2.0/foreign-key.md

```diff
@@ -343,6 +349,232 @@ Let's check to make sure the rows in `orders_2` where `customers_id = 23` were a
 +-----+--------------+
 ~~~
 
+### Use a Foreign Key Constraint with `SET NULL` <span class="version-tag">New in v2.0</span>
+
+In this example, we'll create a table with a foreign key constraint with the [foreign key actions](#foreign-key-actions-new-in-v2-0) `ON UPDATE SET NULL` and `ON DELETE SET NULL`.
+
+First, create the referenced table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE customers_3 (
+    id INT PRIMARY KEY
+  );
+~~~
+
+Then, create the referencing table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE orders_3 (
+    id INT PRIMARY KEY,
+    customer_id INT REFERENCES customers_3(id) ON UPDATE SET NULL ON DELETE SET NULL
+  );
+~~~
+
+Insert a few records into the referenced table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> INSERT INTO customers_3 VALUES (1), (2), (3);
+~~~
+
+Insert some records into the referencing table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> INSERT INTO orders_3 VALUES (100,1), (101,2), (102,3), (103,1);
+~~~
+
+Now, let's update an `id` in the referenced table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> UPDATE customers_3 SET id = 23 WHERE id = 1;
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM customers_3;
+~~~
+~~~
++----+
+| id |
++----+
+|  2 |
+|  3 |
+| 23 |
++----+
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM orders_3;
```

Comment:
It might be nice to show the before and after for these.

Reply:
Done